src/main/asciidoc/reference/migration-guide-2.2-to-3.0.adoc

```diff
@@ -182,9 +182,9 @@ Keyspace creation via `CqlSessionFactoryBean` (`cassandra:session`) is not affec
 
 * `GuavaListenableFutureAdapter`
 * `QueryOptions` and `WriteOptions` constructor taking `ConsistencyLevel` and `RetryPolicy` arguments.
```

Comment:
We need to stick with `executionProfile` as that's part of the Cassandra driver API: https://docs.datastax.com/en/drivers/java/4.6/com/datastax/oss/driver/api/core/cql/StatementBuilder.html#executionProfile